proto/interchain_security/ccv/child/v1/child.proto

```diff
@@ -3,15 +3,27 @@ package interchain_security.ccv.child.v1;
 
 option go_package = "github.com/cosmos/interchain-security/x/ccv/child/types";
 
+import "gogoproto/gogo.proto";
+
+// CrossChainValidator defines the validators for CCV child module
+message CrossChainValidator {
+    bytes address = 1;
+    int64 power = 2;
+}
 
 // Params defines the parameters for CCV child module
 message Params {
     bool Enabled = 1;
-}
 
+    // distribution params
+    int64  BlocksPerDistributionTransmission = 2; 
+    string DistributionTransmissionChannel = 3; 
+    string ProviderFeePoolAddrStr = 4; 
+    string ConsumerRedistributeFrac = 5;  // ex: "0.5"
```

Comment:
Add a comment to each one explaining what it is. This would be useful especially for `ConsumerRedistributeFrac`.